Jenkinsfile

```diff
@@ -326,6 +326,14 @@ timestamps {
 
 					// TODO parallelize the iOS/Android/Windows portions?
 					ansiColor('xterm') {
+						timeout(15) {
+							def buildCommand = "npm run clean -- --android-ndk ${env.ANDROID_NDK_R16B} --android-sdk ${env.ANDROID_SDK}"
+							if (includeWindows) {
+								buildCommand += ' --all'
+							}
+							sh buildCommand
+							recordIssues(tools: [clang(), java()])
```

Comment:
I believe the build is failing due to the:
```Jenkinsfile
recordIssues(tools: [clang(), java()])
```
Removing this line should resolve the issue.